src/AudioPlayer.cpp

```diff
@@ -1067,21 +1060,17 @@ size_t AudioPlayer_NvsRfidWriteWrapper(const char *_rfidCardId, const char *_tra
 
 // Adds webstream to playlist; same like SdCard_ReturnPlaylist() but always only one entry
 char **AudioPlayer_ReturnPlaylistFromWebstream(const char *_webUrl) {
-	char *webUrl = x_strdup(_webUrl);
-	static char **url;
+	static char number[] = "1";
+	static char *url[2] = {number, NULL};
 
-	if (url != NULL) {
-		--url;
-		freeMultiCharArray(url, strtoul(*url, NULL, 10));
+	if(url[1] != NULL) {
```

Comment:
`url[1]` is never NULL here, right?

Reply:
No, on the first call of the function, `url[1]` is pointing to `NULL`. But yes, The check here is not needed, since [free ](https://en.cppreference.com/w/c/memory/free) does nothing, if the supplied pointer is a `nullptr`.